### PR TITLE
[CFM-1178] Allow visitors to send a copy on the global contact form.

### PR DESCRIPTION
**Pull request by ordavidil**
https://issuetracker.amplexor.com/jira/browse/CFM-1178

### CODE REVIEW

**Comment by ybaras** [comment]
**text/html:**

screenshot from 2016-10-10 13 44 40

**Comment by ybaras** [comment]
**Visitor (no WYSIWYG)**

screenshot from 2016-10-10 16 53 05

**Registered user (with editor)**

screenshot from 2016-10-10 16 54 29

File: project/profiles/capacity4more/modules/c4m/features/c4m_contact/c4m_contact.module [hunk 1]
```diff
@@ -51,6 +51,8 @@ function c4m_contact_c4m_captcha_form_info() {
 function c4m_contact_mail_alter(&$message) {
   if ($message['id'] == 'contact_page_mail') {
     $message['to'] = variable_get('c4m_contact_email_addresses', variable_get('site_mail'));
+    $message['headers']['Content-Type'] = 'text/html; charset=UTF-8; format=flowed;';
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features/c4m_contact/c4m_contact.module]: Do we still need this one ?

File: project/profiles/capacity4more/modules/c4m/features/c4m_contact/c4m_contact.module [hunk 2]
```diff
@@ -90,6 +98,21 @@ function c4m_contact_form_contact_site_form_alter(&$form, &$form_state, $form_id
...
+/**
+ * Submit handler for the site contact form.
+ *
+ * Keeps the value only from the data array received from a WYSIWYG field.
+ * Inserts HTML line breaks before all newlines in the value from a textarea.
+ */
+function c4m_contact_contact_site_form_submit($form, &$form_state) {
+  if (isset($form_state['values']['message']['value'])) {
+    $form_state['values']['message'] = $form_state['values']['message']['value'];
+  } else {
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features/c4m_contact/c4m_contact.module]: `else` should be in a new line

File: project/profiles/capacity4more/modules/c4m/features/c4m_contact/c4m_contact.module [hunk 3]
```diff
@@ -90,6 +98,21 @@ function c4m_contact_form_contact_site_form_alter(&$form, &$form_state, $form_id
       '#weight' => -1,
     );
   }
+  array_unshift($form['#submit'], 'c4m_contact_contact_site_form_submit');
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features/c4m_contact/c4m_contact.module]: Add a comment

**Comment by ordavidil** [comment]
@ybaras 
Did you check also the _self copy_ as well ? (for both roles)

**Comment by ybaras** [comment]
@ordavidil Yes, both roles. No problems.

**Pull request merged**